Source/Core/Core/CoreParameter.h

```diff
@@ -201,6 +204,15 @@ struct SCoreStartupParameter
 	int iHotkey[NUM_HOTKEYS];
 	int iHotkeyModifier[NUM_HOTKEYS];
 
+	//Remove Layer
+	std::vector<SkipEntry> object_removal_codes;
+	u32 skip_objects_end = 0;
+	u32 skip_objects_start = 0;
+	size_t num_object_removal_codes = 0;
+	std::vector<int> num_object_removal_data_bytes;
+	volatile bool update = true;
+	volatile bool done = true;
```

Comment:
@degasus Yeah, I'm aware of its existence, though I've never used it before.  I'm sure I can figure it out by googling a bit.  Do you know what the speed implications of std::mutex are?  If I replace the bool locks with a mutex lock, would it be slower?  I was concerned about the speed of the locks when I was first writing this code, but maybe I was worrying for no reason.